os/board/rtl8721csm/src/component/common/drivers/wlan/realtek/include/wifi_constants.h

```diff
@@ -53,7 +53,9 @@ extern "C" {
 #define WPA3_SECURITY		0x00800000
 #define WPS_ENABLED     0x10000000
 
-#define RTW_MAX_PSK_LEN		(64)
+#define RTW_WPA2_MAX_PSK_LEN		(64)

```

Comment:
Is this define related to `IW_WPA2_PASSPHRASE_MAX_SIZE`?
If then, how about defining the value with `IW_WPA2_PASSPHRASE_MAX_SIZE` instead of `64`?

Reply:
Dear @jeongchanKim ,
This is different, RTW_WPA2_MAX_PSK_LEN is defined for upper layer.